astroquery/skyview/tests/test_skyview_remote.py

```diff
@@ -39,7 +40,7 @@ def setup_class(cls):
                              zip(survey_dict.keys(), survey_dict.values()))
     def test_survey(self, survey, survey_data):
         # The print should help discover changes
-        print("Survey: {0} \n Canned reference return: {1} \n"
+        log.info("Survey: {0} \n Canned reference return: {1} \n"
```

Comment:
this can stay as a print, as the comment above indicates this only serves as a help for the maintainers during failing test (this remote test regularly fails as the remote service changes quite often)